data/transactions/logic/eval.go

```diff
@@ -774,6 +771,41 @@ func opAddw(cx *evalContext) {
 	cx.stack[last].Uint = sum
 }
 
+func uint128(hi uint64, lo uint64) *big.Int {
+	whole := new(big.Int).SetUint64(hi)
```

Comment:
it is interesting why does not `big` package like stack-allocated `Int`s... Have you tried
```
whole := &big.Int{}
whole.SetUint64(hi)
...
```
?

Reply:
I think that's fine too, but why prefer two lines / statements instead of a single expression?

I wanted to use syntax that gave a single expression that produced a big.Int from a u64, especially since I needed similar syntax two lines later to create the low word.